.gitignore

```diff
@@ -1 +1,2 @@
 .env
+__pycache__
```

Comment:
Looks like the pycache folder still made it to the PR. Perhaps try /__pycache__. If you're doing this in VSCode, then after you get it right, the folder should gray out. No biggie tho